metrics/examples/grafana/strimzi-kafka.json

```diff
@@ -1426,7 +1426,7 @@
       ],
       "datasource": "${DS_PROMETHEUS}",
       "description": "Incoming Messages Rate",
-      "format": "none",
+      "format": "wps",
```

Comment:
out of curiosity what's "wps" I didn't find it in the doc.

Reply:
Writes per second.

Reply:
Unfortunately there seems to be no _messages per second`, but this seemed kind of suitable.